tests/headers/issue-848/an-include.h

```diff
@@ -0,0 +1,12 @@
+template<typename T>
+class nsTArray {
+  void* mBuffer;
+};
+
+/**
+ * <div rustbindgen replaces="nsTArray"></div>
```

Comment:
*Super* happy you were able to create a regression test.

For posterity, I'd appreciate a comment here about how comments aren't parsed by clang for system headers, and how that breaks our annotations. Basically a summary of all the discoveries that ended up in #848's discussion.